pycc/tests/test_032_localccd.py

```diff
@@ -48,7 +48,7 @@ def test_pnopp_ccd():
                       'd_convergence': 1e-13,
                       'r_convergence': 1e-13,
                       'diis': 1})
-    mol = psi4.geometry(moldict["(H2O)_2"])
+    mol = psi4.geometry(moldict["H2O"])
```

Comment:
I have fixed the issue with PNO++. It was the same issue with the ccd case where I needed to make sure the phases of the eigenvectors were symmetric.

Reply:
Can you be more specific as to what issue you're referring?

Reply:
When I tried running PNO++ with H2O and 6-31G, it would have an error of 1e-3 compared to the simulation code. I thought it had something to do with the perturbed pair density like what we discussed on Monday but it ended up just being the phase factors of `Q[ij]` and `L[ij]` not matching with `Q[ji]`and L[ji]'.

Reply:
OK, good.  I thought that's what you were referring to, but I wanted to make sure.  Are all issues settled now?

Reply:
Yes, everything is good to go.